src/components/StakingVaults/AllEarnOpportunities.tsx

```diff
@@ -29,7 +29,7 @@ export const AllEarnOpportunities = () => {
 
   const sortedVaults = useSortedVaults()
 
-  const { data: foxyBalancesData } = useFoxyBalances()
+  const { data: foxyBalancesData } = useFoxyBalances({ accountNumber: 0 })
```

Comment:
When we add enumeration, `useFoxyBalances` will eventually be able to return all balances, so we will be able to ditch this